fv3core/pace/fv3core/stencils/d_sw.py

```diff
@@ -94,8 +94,6 @@ def heat_diss(
         ke_bg (in):
     """
     with computation(PARALLEL), interval(...):
-        heat_source = 0.0
-        diss_est = 0.0
```

Comment:
These lines need to not be deleted XD

Reply:
I know. But this time GitHub will stop, because I rebased what was making my life hell

Reply:
Done